lib/cluster/vpc.rb

```diff
@@ -53,6 +53,24 @@ def self.find_existing
       end
     end
 
+    def self.create_flowlog
```

Comment:
VPC flow logs let you analyze the ACCEPT/REJECT actions on all traffic passing through the VPC